app/src/main/java/org/jellyfin/androidtv/util/profile/ProfileHelper.kt

```diff
@@ -121,7 +121,7 @@ object ProfileHelper {
 		}
 	}
 
-	fun audioDirectPlayProfile(vararg containers: String) = DirectPlayProfile()
+	fun audioDirectPlayProfile(containers: Array<String>) = DirectPlayProfile()
```

Comment:
Not entirely sure about this change, you can deconstruct an array to vararg with `*`. From Kotlin docs:

```kotlin
fun foo(vararg strings: String) { /*...*/ }

foo(strings = *arrayOf("a", "b", "c"))
```

https://kotlinlang.org/docs/functions.html

Reply:
I originally did this, but Detekt flagged the spread operator as a potentially expensive [operation](https://detekt.github.io/detekt/performance.html#spreadoperator)
Their conformant example shifts the `arrayOf` back into the caller, or elides the spread operator so I was a bit unsure what to do and opted for concatenating two arrays in 761b35851cafee9e3336903ef063cdc00877d379

Reply:
Honestly, for a function that isn't called that often and the array being extremely small I don't believe it impacts us that much. But we can keep this new approach 👍.